app/views/components/gds/exportsInputTextArea.scala.html

```diff
@@ -14,19 +14,25 @@
  * limitations under the License.
  *@
 
+@import scala.collection.immutable
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 @import views.components.gds.Styles._
+@import views.helpers.InputTextHelper
+@import views.html.components.gds.label
+@import play.twirl.api.HtmlFormat
 
-@this(govukTextarea : GovukTextarea)
+@this(govukTextarea : GovukTextarea, label: label, helper: InputTextHelper)
 
 @(
-        field: Field,
-        labelKey: String,
-        hintKey: Option[String] = None,
-        isPageHeading: Boolean = false,
-        headingClasses: String = gdsPageLabel,
-        labelClasses: String = "govuk-label--m",
-        inputClasses: Option[String] = None
+    field: Field,
+    labelKey: String,
+    labelArg: String = "",
+    hintKey: Option[String] = None,
+    bodyHtml: Option[Html] = None,
```

Comment:
Could this be renamed to something like `bodyUnderLabel`, `paragraphUnderLabel` or similar so to make clear what's for?